device-types/Softline/softline_sl911.yaml

```diff
@@ -0,0 +1,31 @@
+---
+manufacturer: Softline
+model: Softline_SL911
+slug: softline_sl911
+u_height: 2
+weight: 20
+weight_unit: kg
+is_full_depth: true
+airflow: front-to-rear
+comments: '2U 8x3.5 bay Storage, Inwin RS208-02M-CR550.H case'
```

Comment:
Have you got a spec sheet for this. if this is just a Inwin RS208-02M-CR550.H then it probably should be modeled as one of those